### PR TITLE
Fix date sorting

### PR DESCRIPTION
**Pull request by ericljpemberton**
Just had to change from datasort to dataSort, also fixed a spacing thing

### CODE REVIEW

**Comment by ericljpemberton** [comment]
@schneider-clarisse

File: src/js/pages/organizer/Calendar.jsx [hunk 1]
```diff
@@ -1,29 +1,112 @@
...
+    this.updateData = this.updateData.bind(this);
+    console.log(props);
+    this.state = {
+      stream: props.defaultValue.stream,
+      streams: props.streams,
+     };
+  }
+
+  focus() {
+    console.log("HERE");
```
Comment (larajanecka) [inline src/js/pages/organizer/Calendar.jsx:20]: should this be here?

File: src/js/pages/organizer/Calendar.jsx [hunk 2]
```diff
@@ -1,29 +1,112 @@
...
+  updateData(ev) {
+    console.log("UPDATE DATA");
+    console.log(ev.target.value);
+    // this.props.onUpdate({ stream: ev.target.value });
+    this.setState({ stream: ev.currentTarget.value });
+  }
+
+  render() {
+
+    const divStyle = {
```
Comment (larajanecka) [inline src/js/pages/organizer/Calendar.jsx:33]: no inline styling

File: src/js/pages/organizer/Calendar.jsx [hunk 3]
```diff
@@ -1,29 +1,112 @@
...
+          value={ this.state.stream }>
+          { this.state.streams.map(opt => (<option key={opt} value={ opt }>{ opt }</option>)) }
+        </select>
+      </span>
+    );
+  }
+}
+
+/*
+      <span>
```
Comment (larajanecka) [inline src/js/pages/organizer/Calendar.jsx:55]: kill this

File: src/js/pages/organizer/Calendar.jsx [hunk 4]
```diff
@@ -1,29 +1,112 @@
...
 var moment = require('moment');
 
+
+class StreamEditor extends React.Component {
+  constructor(props) {
+    super(props);
+    this.updateData = this.updateData.bind(this);
+    console.log(props);
+    this.state = {
+      stream: props.defaultValue.stream,
```
Comment (larajanecka) [inline src/js/pages/organizer/Calendar.jsx:14]: this feels odd. why the default value over the value? I dont see where the default value is defined.

**Comment by ericljpemberton** [comment]
TOTALLY MY BAD, second commit should not be a part of this...

**Pull request closed without merging**